cards/src/getUsers.js

```diff
@@ -0,0 +1,19 @@
+import { UserCard } from "./UserCard";
+
+async function getUsers(userID) {
+    
+    const url = `https://reqres.in/api/users/${userID}`;
```

Comment:
This function is well put together - easy to read, and having the url variable is a nice touch. 👍 👍